local-cli/runWindows/utils/build.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const execSync = require('child_process').execSync;
+const path = require('path');
+const chalk = require('chalk');
+const glob = require('glob');
+const shell = require('shelljs');
+const MSBuildTools = require('./msbuildtools');
+
+function cleanSolution(options) {
+  const appPackagesPath = glob.sync(path.join(options.root, 'windows/*/AppPackages'));
+  console.log(chalk.green(`Cleaning AppPackages folder ${appPackagesPath}`));
+  shell.rm('-rf', appPackagesPath);
+}
+
+function buildSolution(slnFile, buildType, buildArch) {
+  console.log(chalk.green(`Building ${slnFile}`));
+  const msBuildTools = MSBuildTools.findAvailableVersion();
+  msBuildTools.buildProject(slnFile, buildType, buildArch, null);
+}
+
+function restoreNuGetPackages(options, slnFile) {
+  console.log(chalk.green('Restoring NuGet packages'));
+  const nugetPath = options.nugetPath || path.resolve('../.nuget/nuget.exe');
```

Comment:
I still don't think this path is right.

---
In reply to: [68117303](https://github.com/ReactWindows/react-native-windows/pull/464#discussion_r68117303) [](ancestors = 68117303)